src/status_im/ui/components/toolbar/styles.cljs

```diff
@@ -70,7 +70,7 @@
 
 (def nav-item-button
   {:padding-vertical   16
-   :padding-horizontal 12})
+   :padding-horizontal 120})
```

Comment:
Is this intentional?

Reply:
oops, leftover from testing. i'll fix that